cmd/common/prompts.go

```diff
@@ -33,13 +38,20 @@ func Confirm(msg, abortMsg string) {
 
 	var proceed bool
 	err := survey.AskOne(&survey.Confirm{Message: msg}, &proceed)
-	cobra.CheckErr(err)
+	if err != nil {
+		if errors.Is(err, io.EOF) {
```

Comment:
Assuming yes to all for any non-interactive TTY seems dangerous. Can you just run your tests with `-y` flag?

Reply:
thanks for catching! done